### DISCUSSION TITLE
求PC网页端写日记时添加默认标签的功能

### DISCUSSION THREAD

**Discussion by yyysuo** [usememos/memos#63] [Ideas]
ios端可以通过快捷指令根据剪贴板和手工输入的方式给日记打标签，挺好用的。想在pc端输入日记的时候也打上默认的标签，求添加此功能。
![image](https://user-images.githubusercontent.com/10081071/169480139-5625596d-b966-4e50-a71e-79f6a5bb543f.png)

![281601969926182353](https://user-images.githubusercontent.com/10081071/169480186-0ab50fc7-e8b7-4ef3-8ce5-bcfa80612581.jpg)

![798383497504094348](https://user-images.githubusercontent.com/10081071/169480223-8af567af-b1c4-4cde-9547-c8a0cd32811d.jpg)

**Comment by boojack** [comment]
建议通过 shortcut 来实现
<img width="539" alt="image" src="https://user-images.githubusercontent.com/24653555/169497662-f08d0abd-7394-4efe-a95b-be8c2b4ef2ec.png">

**Reply by yyysuo** [reply]
> 建议通过 shortcut 来实现 <img alt="image" width="539" src="https://user-images.githubusercontent.com/24653555/169497662-f08d0abd-7394-4efe-a95b-be8c2b4ef2ec.png">

感谢，发现了新大陆，确实能部分实现我的需求，只不过是不是标签，也不能用图标呀，不够漂亮。

**Reply by boojack** [reply]
在 Title 里自行加上 emoji？
<img width="284" alt="image" src="https://user-images.githubusercontent.com/24653555/169515400-0d2235b0-e97d-45ba-a952-7a9db0efb337.png">

**Comment by yyysuo** [comment]
求具体的加法。

STEVEN ***@***.***>于2022年5月20日 周五下午7:06写道：

> 在 Title 里自行加上 emoji？
> [image: image]
> <https://user-images.githubusercontent.com/24653555/169515400-0d2235b0-e97d-45ba-a952-7a9db0efb337.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/justmemos/memos/discussions/63#discussioncomment-2790193>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACM5GL6UNX4MBCGQBQOXAJ3VK5W43ANCNFSM5WOS56UA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
马志浩

**Reply by boojack** [reply]
纯文本输入框里键入“💻 ”
<img width="538" alt="image" src="https://user-images.githubusercontent.com/24653555/169516454-a6b631a4-e2da-4ea6-b535-d52109f553a8.png">

**Comment by yyysuo** [comment]
我用iPhone通过微信的文件助手给电脑端的微信发了一个电脑图标 复制之后粘贴不到文本框里


STEVEN ***@***.***>于2022年5月20日 周五下午7:15写道：

> 纯文本输入框里键入“💻 ”
> [image: image]
> <https://user-images.githubusercontent.com/24653555/169516454-a6b631a4-e2da-4ea6-b535-d52109f553a8.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/justmemos/memos/discussions/63#discussioncomment-2790244>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACM5GL6NYHQQQHQWLHUE6STVK5X55ANCNFSM5WOS56UA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
马志浩
